dist/index.js

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env node
-
-require('./cli');
```

Comment:
Pretty sure we need to keep this actually, forgot about that. CLI won't work without it.